<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1689132069801' stop='1689132071986'>
    <name>Faker:</name>
    <title>Faker:</title>
    <test-cases>
        <test-case start='1689132070908' status='failed' stop='1689132071707'>
            <name>FakerDemo</name>
            <title>FakerDemo</title>
            <labels>
                <label name='language' value='javascript'/>
                <label name='framework' value='codeceptjs'/>
            </labels>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Main session - Last Seen Screenshot' source='51167d35-8f3c-4f96-b2b3-734cd0c6a139-attachment.png' type='image/png' size='6897'/>
            </attachments>
            <failure>
                <message>faker1.name.findName is not a function</message>
                <stack-trace>TypeError: faker1.name.findName is not a function
                    at Test.&lt;anonymous> (C:\POC\tests\faker.js:17:34)
                    at test.fn (C:\POC\node_modules\codeceptjs\lib\scenario.js:92:14)
                    at callFnAsync (C:\POC\node_modules\mocha\lib\runnable.js:394:21)
                    at Runnable.run (C:\POC\node_modules\mocha\lib\runnable.js:338:7)
                    at Runner.runTest (C:\POC\node_modules\mocha\lib\runner.js:666:10)
                    at C:\POC\node_modules\mocha\lib\runner.js:789:12
                    at next (C:\POC\node_modules\mocha\lib\runner.js:581:14)
                    at C:\POC\node_modules\mocha\lib\runner.js:591:7
                    at next (C:\POC\node_modules\mocha\lib\runner.js:474:14)
                    at cbHookRun (C:\POC\node_modules\mocha\lib\runner.js:539:7)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>